test/app/core/usecase/test_trading_usecase.py

```
@@ -106,31 +106,40 @@ def sample_ticker(self):
         )
 
     @pytest.mark.asyncio
-    async def test_execute_trading_algorithm_simulation_mode(
+    async def test_execute_trading_algorithm_live_mode(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)